moodle/Util/MoodleUtil.php

```diff
@@ -16,6 +16,7 @@
 
 namespace MoodleHQ\MoodleCS\moodle\Util;
 
+use Exception;
```

Comment:
Not asking for any change on this, just sharing that it's the exact type of `use` use (heh), that always keep me thinking about what's better. To use or not to use (global scope classes), that's the question.

Again, not asking for changing it. At all!

Reply:
This is one of those cases of the IDE getting trigger happy. Fixed.